app/grants/templates/grants/fund.html

```diff
@@ -0,0 +1,137 @@
+{% load static %}
+{% load humanize %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  {% include 'shared/head.html' %}
+  {% include 'shared/cards.html' %}
+
+</head>
+<body class="interior {{active}} grant">
+{% include 'shared/tag_manager_2.html' %}
+<div class="container-fluid header dash">
+  {% include 'shared/nav.html' %}
+</div>
+
+<img src="{% static "v2/images/bitmap.png" %}" width="100%" />
+
+<h2 class="fund__header">
+  Help
+  <span class="fund__header--highlight">{{grant.adminProfile}}</span>
+  get funding for
+  <span class="fund__header--highlight">{{grant.title}}</span>
+</h2>
+
+<div class="container">
+  <div class="fund__title">Fund Grant</div>
+
+  <form action='{{grant.pk}}' method="POST" id="js-fundGrant">
+    {% csrf_token %}
+    <div class="row bottom-buffer">
+
+
+
+      <div class="form__group-horizontal">
+        <label class="form__label">Contribution per Period</label>
```

Comment:
We'll want to wrap all copy throughout these django templates with `{% trans "Copy" %}`